test_crates/inherent_associated_const_now_doc_hidden/new/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+publish = false
+name = "inherent_associated_const_now_doc_hidden"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
```

Comment:
nit: It's usually customary to end files with a trailing newline, since that makes sure uses with other tools work properly. I think you were using VSCode, right? If so, you can make it automatically add the trailing newline by applying the following setting: `"files.insertFinalNewline": true`
```suggestion
[dependencies]

```